exporters/prometheus/src/module/java/module-info.java

```diff
@@ -0,0 +1,7 @@
+@SuppressWarnings({"requires-automatic", "requires-transitive-automatic"})
+module io.opentelemetry.exporter.prometheus {
+  exports io.opentelemetry.exporter.prometheus;
+
+  requires transitive io.opentelemetry.sdk.metrics;
```

Comment:
prometheus simpleclient doesn't define an automatic module name so doesn't work here, luckily that codepath is deprecated now